room/views.py

```diff
@@ -1,3 +1,97 @@
-from django.shortcuts import render
+from django.views     import View
+from django.http      import JsonResponse
+from django.db.models import Avg
 
-# Create your views here.
+from room.models import Room, Image, RoomConvenience, RoomRule, Review
+from user.models import User, Host
+
+class RoomDetailView(View):
+    def get(self, request, room_id):
+        checkin_date  = request.GET.get('check_in', None)
+        checkout_date = request.GET.get('check_out', None)
+        adults        = request.GET.get('adults', 0) 
+        child         = request.GET.get('child', 0) 
+        baby          = request.GET.get('baby', 0) 
+
+        exist_room = Room.objects.filter(id=room_id).exists()
+        if not exist_room:
+            return JsonResponse({'message': 'ROOM_DOES_NOT_EXIST'}, status=404)
+
+        room = Room.objects.get(id=room_id)
+
+        result = {
+                "checkin_date": checkin_date,
+                "checkout_date": checkout_date,
+                "adult": adults,
+                "child": child,
+                "baby": baby,
+                "number_of_guests": int(adults)+int(child)+int(baby),
+                "price": room.price,
+                "discount_rate": room.discount_rate,
+                "title": room.title,
+                "city": room.city.name,
+                "district": room.district.name,
+                "address_line": room.address_line.name,
+                "images_url": [image.image_url for image in Image.objects.filter(room=room)],
```

Comment:
이것도 
```suggestion
                "images_url": [image.image_url for image in room.image_set.all()],
```
이렇게 하면 되지 않을까요?